core/kernel.rbs

```diff
@@ -614,7 +617,7 @@ module Kernel : BasicObject
                    | (int | _ToI int_like, exception: bool) -> Integer?
                    | (string str, int base, ?exception: true) -> Integer
                    | (string str, int base, exception: bool) -> Integer?
-                   | (untyped, ?untyped, ?exception: bool) -> Integer?
+                   | (untyped, ?int base, exception: bool) -> Integer?
```

Comment:
If provided, the second argument actually has to be an `int`, otherwise an error will be thrown (regardless of `exception`)